Eventy/src/main/java/org/example/eventy/solutions/controllers/ServiceReservationController.java

```diff
@@ -0,0 +1,78 @@
+package org.example.eventy.solutions.controllers;
+
+import org.example.eventy.solutions.dtos.ServiceReservationDTO;
+import org.example.eventy.solutions.dtos.SolutionCardDTO;
+import org.example.eventy.solutions.services.ServiceReservationService;
+import org.example.eventy.solutions.services.ServiceService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+@RestController
+@RequestMapping("api/reservations")
+public class ServiceReservationController {
+
+    @Autowired
+    private ServiceReservationService serviceReservationService;
+
+    /*
+    {
+        "id": 5,
+        "selectedEventId": 3,
+        "selectedSolutionId": 12,
+        "reservationStartDateTime": "2024-12-01T14:00:00",
+        "reservationEndDateTime": "2024-12-01T16:00:00"
+    }
+    */
+    // POST "/api/reservations"
+    @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Is there any other kind of reservation? Because then this isnt probably the best name, but if there isnt this is perfect!!

Reply:
I am still not sure, but I can rename it if needed